src/test/resources/expected-classes/optional/NodeQueryResolver_mono.java.txt

```diff
@@ -0,0 +1,10 @@
+
+@javax.annotation.Generated(
+    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
+    date = "2020-12-31T23:59:59-0500"
+)
+public interface NodeQueryResolver {
+
+    reactor.core.publisher.Mono<java.util.Optional<Node>> node(String id) throws Exception;
```

Comment:
Mono itself can describe 0 or 1 in reactor.Users of the stream should be aware of this.

Reply:
Yes, you are right. But have you seen the bug description? There could be some types that can expect Optional as a sub-type. Anyway, this Optional thing can be disabled. That's up to the client to decide.